src/FlowInstancesPage.tsx

```diff
@@ -152,7 +152,11 @@ export default (): React.ReactElement => {
       >
         {(instances) => <InstancesTable instances={instances} onDelete={handleDeleteFlow} />}
       </WaitForData>
-      <Button variant="primary" onClick={() => navigate('/pipelines')}>
+      <Button
+        variant="primary"
+        hidden={!astarte.token?.can('flow', 'GET', '/pipelines')}
+        onClick={() => navigate('/pipelines')}
+      >
```

Comment:
We also want to edit the TableRow component to
- display a label instead of the link to the Flow Details page when
  ```
  !astarte.token?.can('flow', 'GET', '/flows/${instance.name}')
  ```
- hide the "Delete instance" button when
  ```
  !astarte.token?.can('flow', 'DELETE', '/flows/${instance.name}')
  ```